HHS/Head-Start-TTADP
# [NO-TICKET] Make sure courses can be saved

## Description of change
Looks like recent changed introduced a bug where courses can't be saved. This PR is a very small code change to fix that

## How to test
On the AR, add a course to an objective. Save the objective. Note that the course appears in the "read-only" view (with the bug, it would not). Edit the objective, and you will see the course is still there.

## Issue(s)

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
